client/src/app/tabs/dmn/DmnEditor.js

```diff
@@ -749,6 +747,8 @@ export class DmnEditor extends CachedComponent {
       error = _error;
     }
 
+    const stackIdx = modeler.getStackIdx();
```

Comment:
We always save the stack index even in the case of an error. We don't do that for BPMN. 🤔

Reply:
I noticed that as well, but decided to not touch it for now. Because it was this way before, I would consider it out of scope for this ticket.
Of course, we should investigate if this is causes a problem and create a follow-up for it.

Reply:
Okay, then let's have a follow-up issue for it. The code is being changed without any test verifying what the change does.

Reply:
You're right, I forgot to change the dmn-js mock. I'll fix that

Reply:
👍🏻

Reply:
I aligned the DMN save with BPMN in 756dfb5be2f32b9df481130416cf3806d38060c2